test/test_bill_com/test_bill_com.py

```diff
@@ -0,0 +1,265 @@
+                                                     'payToBankAccountId': '0',
+                                                     'payToChartOfAccountId': '0',
+                                                     'invoiceTemplateId': '0',
+                                                     'hasAutoPay': False,
+                                                     'source': '0',
+                                                     'emailDeliveryOption': '1',
+                                                     'mailDeliveryOption': '0',
+                                                     'creditAmount': 0.0,
+                                                     'invoiceLineItems': [{'entity':
+                                                                           'InvoiceLineItem',
+                                                                           'id':
+                                                                           'fake_line_item_id',
+                                                                           'createdTime':
+                                                                           self.fake_time,
+                                                                           'updatedTime':
+                                                                           self.fake_time,
+                                                                           'invoiceId':
+                                                                           'fake_invoice_id',
+                                                                           'itemId': '0',
+                                                                           'quantity': 1,
+                                                                           'amount': 1.0,
+                                                                           'price': None,
+                                                                           'serviceDate': None,
+                                                                           'ratePercent': None,
+                                                                           'chartOfAccountId': '0',
+                                                                           'departmentId': '0',
+                                                                           'locationId': '0',
+                                                                           'actgClassId': '0',
+                                                                           'jobId': '0',
+                                                                           'description': None,
+                                                                           'taxable': False,
+                                                                           'taxCode': 'Non'}]}]}
+
+        self.fake_invoice_read_json = self.fake_invoice_read_json = \
```

Comment:
This looks like a copy/paste gone wrong (probably my most frequent type of error). I can just fix it in branch.